plugin.video.frikanalen/frikanalen.py

```diff
@@ -90,7 +142,34 @@ def whats_on():
     for item in program:
         t = item.starttime
         if t.hour == now.hour:
-            print("> [{:d}:{:02d}] {:s} {:s}".format(t.hour, t.minute, item.video.name, item.duration))
+            print("> [{:d}:{:02d}] {:s} {:f}".format(t.hour, t.minute, item.video.name, item.duration))
```

Comment:
Please use xbmc.log instead of print statements

Reply:
This print code is so far only used by the self test, which I run outside Kodi.  It is a goal to keep the frikanalen.py code workout without any xbmc dependencies to allow the API access to be tested without having the xbmc* libraries installed.  The print lines will be changed/removed when this code is called in the addon code path.